node/sc/sub_bridge_handler.go

```diff
@@ -342,6 +376,39 @@ func (sbh *SubBridgeHandler) LocalChainHeadEvent(block *types.Block) {
 	}
 }
 
+func (sbh *SubBridgeHandler) handleParentChainInvalidTxResponseMsg(msg p2p.Msg) error {
```

Comment:
This is a bit confusing, main - sub vs. parent - child.
Are they different concepts?

Reply:
So called, 
Bridge
- Main Bridge <-> Sub Bridge

Chain
- Parent Chain <-> Child Chain
- Main Chain <-> Service Chain

It is also confused to me.

Reply:
@henry-will Thanks.